urns/urns.go

```diff
@@ -178,57 +174,70 @@ func (u URN) Normalize(country string) URN {
 }
 
 // Validate returns whether this URN is considered valid
-func (u URN) Validate() bool {
+func (u URN) Validate() error {
 	scheme, path, display := u.ToParts()
 	if !IsValidScheme(scheme) || path == "" {
-		return false
+		return fmt.Errorf("invalid path: '%s'", path)
 	}
 
 	switch scheme {
 	case TelScheme:
 		// validate is possible phone number
 		parsed, err := phonenumbers.Parse(path, "")
 		if err != nil {
-			return false
+			return err
+		}
+		if !phonenumbers.IsPossibleNumber(parsed) {
+			return fmt.Errorf("impossible phone number: %s", path)
```

Comment:
I don't think we want to set the bar this high for phone numbers.. it's ok to accept a phone number that the phonenumbers library doesn't know about (as long as it parses ok)